linuxkpi/gplv2/include/linux/dma-buf-map.h

```diff
@@ -0,0 +1,266 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
No new GPLv2 files please.

Reply:
Is there a good method to rewrite such a header?

Reply:
OpenBSD provides a public domain implementation, I will take this one.

Reply:
Done.